packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -87,9 +88,14 @@ const getChartOrDashboard = async (
         const dashboard = await schedulerService.dashboardModel.getById(
             dashboardUuid,
         );
+
         return {
             url: `${lightdashConfig.siteUrl}/projects/${dashboard.projectUuid}/dashboards/${dashboardUuid}/view`,
-            minimalUrl: `${lightdashConfig.siteUrl}/minimal/projects/${dashboard.projectUuid}/dashboards/${dashboardUuid}`,
+            minimalUrl: `${lightdashConfig.siteUrl}/minimal/projects/${
+                dashboard.projectUuid
+            }/dashboards/${dashboardUuid}${
+                schedulerUuid ? `?schedulerId=${schedulerUuid}` : ''
```

Comment:
should param also be `schedulerUuid` as well for consistency?

Reply:
oops! good catch, renaming!